mgmt-hub/deploy-mgmt-hub.sh

```diff
@@ -604,12 +689,20 @@ if isMacOS; then
     sudo installer -pkg $TMP_DIR/pkgs/horizon-cli-*.pkg -target /
     chk $? 'installing macos horizon-cli pkg'
     # we will install the agent below, after configuring /etc/default/horizon
-else   # ubuntu
-    getUrlFile $OH_ANAX_RELEASES/$OH_ANAX_DEB_PKG_TAR $TMP_DIR/pkgs/$OH_ANAX_DEB_PKG_TAR
-    tar -zxf $TMP_DIR/pkgs/$OH_ANAX_DEB_PKG_TAR -C $TMP_DIR/pkgs   # will extract files like: horizon-cli_2.27.0_amd64.deb
-    chk $? 'extracting pkg tar file'
-    echo "Installing the Horizon agent and CLI packages..."
-    runCmdQuietly apt-get install -yqf $TMP_DIR/pkgs/horizon*.deb
+else   # ubuntu and redhat
+    if isUbuntu18; then
+        getUrlFile $OH_ANAX_RELEASES/$OH_ANAX_DEB_PKG_TAR $TMP_DIR/pkgs/$OH_ANAX_DEB_PKG_TAR
+        tar -zxf $TMP_DIR/pkgs/$OH_ANAX_DEB_PKG_TAR -C $TMP_DIR/pkgs   # will extract files like: horizon-cli_2.27.0_amd64.deb
+        chk $? 'extracting pkg tar file'
+        echo "Installing the Horizon agent and CLI packages..."
+        runCmdQuietly ${PKG_MNGR} ${PKG_MNGR_INSTALL_QY_CMD} $TMP_DIR/pkgs/horizon*.deb
+    else # redhat
+        getUrlFile $OH_ANAX_RELEASES/$OH_ANAX_RPM_PKG_TAR $TMP_DIR/pkgs/$OH_ANAX_RPM_PKG_TAR
+        tar -zxf $TMP_DIR/pkgs/$OH_ANAX_RPM_PKG_TAR -C $TMP_DIR/pkgs   # will extract files like: horizon-cli_2.27.0_amd64.rpm
+        chk $? 'extracting pkg tar file'
+        echo "Installing the Horizon agent and CLI packages..."
+        rpm -i --force $TMP_DIR/pkgs/horizon*.rpm
```

Comment:
Why the `--force` flag?

Reply:
The `Horizon` agent and `CLI` packages are installed **always** during run of the `deploy-mgmt-hub.sh` in the installation mode so this flag is mandatory in this case. If I remove this flag `rpm` (or `yum`) installer will return `false (1`) for existing `Horizon RPM` package when we are trying to reinstall `Management  Hub` by some reasons.

Reply:
Fixed. Before installation I am checking Horizon packages and delete ones explicitly if exist.  Close it.